config/SDKVersions.xcconfig

```diff
@@ -4,3 +4,6 @@ _WP_IOS_120000 = XCODE10
 WP_IOS_SDK = $(_WP_IOS_$(SDK_VERSION_MAJOR))
 
 SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(inherited) $(WP_IOS_SDK)
+
+// TODO: Move to dedicated file
+WARNING_CFLAGS = -Wno-nullability-completeness
```

Comment:
Oh! I had forgotten I put the flag in this "not entirely appropriate" config file. 🤔 

I think it's worth merging as is, then followup with some good `.xcconfig` structure.